types/meteor/tracker.d.ts

```diff
@@ -32,7 +32,7 @@ declare module "meteor/tracker" {
 
         function flush(): void;
 
-        function nonreactive(func: Function): void;
+        function nonreactive<T = any>(func: Function): T;
```

Comment:
We could infer the returned type instead - [the implementation simply calls `func`](https://github.com/meteor/meteor/blob/71f67d9dbac34aba34ceef733b2b51c2bd44b665/packages/tracker/tracker.js#L603).
```suggestion
        function nonreactive<T>(func: () => T): T;
```

Reply:
I saw another API that did exactly that - but I have never done it that way before. I'll modify the PR like that once I figured out what to do!